src/System.Diagnostics.Process/tests/ProcessStartInfoTests.netstandard1.7.cs

```diff
@@ -0,0 +1,211 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public partial class ProcessStartInfoTests
+    {
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
It was Windows-specific before doing it a data round trip test because I was testing that case-insensitive cases worked. Will remove the platform specific now. Thanks